users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -391,12 +649,13 @@ def process_args(args: Dict[str, Any]):
         commit="c4d36d06f6465e82a50d400d114259e07b8b0709",
     ).out_repository
     returnn_root.hash_overwrite = "returnn_conv_padding"
-    report, ctc_nn_system = run_nn_args(
+    report = run_nn_args(
```

Comment:
This seems wrong. Did you test this? I suspect if you would start the manager, this would crash.

Reply:
No the manager starts without problems. I will use `report, _` for all experiments so it is consistent

Reply:
Probably only if you start it with the specific experiment function. With the `py()` function I would suspect it to crash.